index.html

```diff
@@ -0,0 +1,44 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Document</title>
+</head>
+<body>
+  <h1 id="title">freeCodeCamp Survey Form</h1>
```

Comment:
Please change the id for a class selector.

Reply:
[OPTIONAL] Good job adding the _why_ 😉😉. I would also recommend you rephrase your requirement making it look like a suggestion, not an order. For example: "I think it would be better to use semantic tags" sounds nicer than "Use semantic tags,"